blackberggroup/VA-OHIL
# Feat/jls 7863937221/add contact us

## What type of PR is this? (check all applicable)


## Description
Add "Contact Us" to the navigation bar.

When clicked, this should open a modal box. The modal should have the following content:
"Contact Us is for VA staff only on the VA network. If you are external to VA, please go to [pathfinder.va.gov](http://pathfinder.va.gov/)"

A button to continue will navigate the user to "[apps.gov.powerapps.us/play/e/default-e95f1b23-abaf-45ee-821d-b7ab251ab3bf/a/2699aa53-7768-446e-817b-d74d107fd4aa?tenantId=e95f1b23-abaf-45ee-821d-b7ab251ab3bf&hint=d8802eae-36cc-443f-a0bd-fdd6034d8196&sourcetime=1724963015938&hidenavbar=true](http://apps.gov.powerapps.us/play/e/default-e95f1b23-abaf-45ee-821d-b7ab251ab3bf/a/2699aa53-7768-446e-817b-d74d107fd4aa?tenantId=e95f1b23-abaf-45ee-821d-b7ab251ab3bf&hint=d8802eae-36cc-443f-a0bd-fdd6034d8196&sourcetime=1724963015938&hidenavbar=true)"

A another button will need to link to [pathfinder.va.gov](http://pathfinder.va.gov/).

## Related Tickets & Documents
- Monday Task https://blackberg-group.monday.com/boards/1931719392/pulses/7863937221

## QA Instructions, Screenshots, Recordings
Upon click, ensure modal works visually and functionally. Left button should open link to VA network. Right button should open PathFinder page. Ensure modal closes upon escape or by clicking close button.

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_


## Are there any post deployment tasks we need to perform?
- Add GA code on production
- Proper meta tags utilized 

## Changed Files in this Pull Request ##
- _includes/header-sub.html
 - _includes/header.html
 - _site/index.html
